### PR TITLE
Finish the lidar use

### PR DESCRIPTION
**Pull request by TsarF**
stupid simple

### CODE REVIEW

**Review by HoldenA1 — COMMENTED** [review]
Get distance function should be lowercase.

File: src/main/java/frc/robot/controllers/DistanceSensor.java [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-    PWM pwmSensor;
+    DigitalInput DIO;
 
-    public DistanceSensor(int PWM_ID)
-    {
-        ID = PWM_ID;
-        pwmSensor = new PWM(ID);
+    Counter counter; // fml if I knew about this before...
+
+    public DistanceSensor(int Channel_ID) {
```
Comment (HoldenA1) [inline src/main/java/frc/robot/controllers/DistanceSensor.java:12]: Function name should be lowercase

File: src/main/java/frc/robot/controllers/DistanceSensor.java [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+        ID = Channel_ID;
+        DIO = new DigitalInput(ID);
+        counter = new Counter(DIO);
     }
 
     public double GetDistance()
```
Comment (HoldenA1) [inline src/main/java/frc/robot/controllers/DistanceSensor.java]: So should this

File: src/main/java/frc/robot/controllers/DistanceSensor.java (hunk 1 above)
Comment (TsarF) [reply src/main/java/frc/robot/controllers/DistanceSensor.java:12]: This is a constructor

File: src/main/java/frc/robot/controllers/DistanceSensor.java (hunk 2 above)
Comment (HoldenA1) [reply src/main/java/frc/robot/controllers/DistanceSensor.java]: Thats an L

**Review by HoldenA1 — APPROVED** [review]
Rip

**Pull request merged**